### DISCUSSION TITLE
Build/DataPipeline/AnalyticsCore

### DISCUSSION THREAD

**Discussion by dikshauat** [Sunbird-Ed/Community#147] [General Discussion]
Hi, 

I am getting the following error while running the job Build/DataPipeline/AnalyticsCore

error: scala.reflect.internal.MissingRequirementError: object java.lang.Object in compiler mirror not found.
[[1;31mERROR[m] 	at scala.reflect.internal.MissingRequirementError$.signal(MissingRequirementError.scala:17)
[[1;31mERROR[m] 	at scala.reflect.internal.MissingRequirementError$.notFound(MissingRequirementError.scala:18)
[[1;31mERROR[m] 	at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:53)
[[1;31mERROR[m] 	at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:45)
[[1;31mERROR[m] 	at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:45)
[[1;31mERROR[m] 	at scala.reflect.internal.Mirrors$RootsBase.getModuleOrClass(Mirrors.scala:66)
[[1;31mERROR[m] 	at scala.reflect.internal.Mirrors$RootsBase.getClassByName(Mirrors.scala:102)
[[1;31mERROR[m] 	at scala.reflect.internal.Mirrors$RootsBase.getRequiredClass(Mirrors.scala:105)
[[1;31mERROR[m] 	at scala.reflect.internal.Definitions$DefinitionsClass.ObjectClass$lzycompute(Definitions.scala:257)
[[1;31mERROR[m] 	at scala.reflect.internal.Definitions$DefinitionsClass.ObjectClass(Definitions.scala:257)
[[1;31mERROR[m] 	at scala.reflect.internal.Definitions$DefinitionsClass.init(Definitions.scala:1390)
[[1;31mERROR[m] 	at scala.tools.nsc.Global$Run.<init>(Global.scala:1242)
[[1;31mERROR[m] 	at scala.tools.nsc.Driver.doCompile(Driver.scala:31)
[[1;31mERROR[m] 	at scala.tools.nsc.MainClass.doCompile(Main.scala:23)
[[1;31mERROR[m] 	at scala.tools.nsc.Driver.process(Driver.scala:51)
[[1;31mERROR[m] 	at scala.tools.nsc.Driver.main(Driver.scala:64)
[[1;31mERROR[m] 	at scala.tools.nsc.Main.main(Main.scala)
[[1;31mERROR[m] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[[1;31mERROR[m] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[[1;31mERROR[m] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[[1;31mERROR[m] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[[1;31mERROR[m] 	at scala_maven_executions.MainHelper.runMain(MainHelper.java:164)
[[1;31mERROR[m] 	at scala_maven_executions.MainWithArgsInFile.main(MainWithArgsInFile.java:26)
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;34mINFO[m] [1mReactor Summary for Sunbird Analytics Framework 2.0:[m
[[1;34mINFO[m] 
[[1;34mINFO[m] Sunbird Analytics Framework ........................ [1;32mSUCCESS[m [  0.312 s]
[[1;34mINFO[m] analytics-core ..................................... [1;31mFAILURE[m [  7.531 s]
[[1;34mINFO[m] Analytics Job Driver - Batch and Streaming ......... [1;33mSKIPPED[m
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;34mINFO[m] [1;31mBUILD FAILURE[m
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;34mINFO[m] Total time:  8.035 s
[[1;34mINFO[m] Finished at: 2022-09-09T16:24:53+05:30
[[1;34mINFO[m] [1m------------------------------------------------------------------------[m
[[1;31mERROR[m] Failed to execute goal [32mnet.alchim31.maven:scala-maven-plugin:3.2.2:compile[m [1m(default)[m on project [36manalytics-core[m: [1;31mwrap: org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)[m -> [1m[Help 1][m
[[1;31mERROR[m] 
[[1;31mERROR[m] To see the full stack trace of the errors, re-run Maven with the [1m-e[m switch.
[[1;31mERROR[m] Re-run Maven using the [1m-X[m switch to enable full debug logging.
[[1;31mERROR[m] 
[[1;31mERROR[m] For more information about the errors and possible solutions, please read the following articles:
[[1;31mERROR[m] [1m[Help 1][m http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[[1;31mERROR[m] 
[[1;31mERROR[m] After correcting the problems, you can resume the build with the command
[[1;31mERROR[m]   [1mmvn <goals> -rf :analytics-core[m
[Pipeline] }

Kindly help me in resolving the error

**Comment by beepdot** [comment]
@sowmya-dixit @anandp504 Can you guys please suggest a solution?

**Reply by dikshauat** [reply]
@sowmya-dixit @anandp504

kindly help us with a solution to the above issue

Thanks

**Reply by sowmya-dixit** [reply]
@dikshauat Can you please let us know what is the tag used for this build job?

**Reply by jaingaurav126** [reply]
we are using release-4.2.0_RC1 for the job
